oidc_provider/views.py

```diff
@@ -174,6 +178,20 @@ def post(self, request, *args, **kwargs):
 
             return redirect(uri)
 
+    @staticmethod
+    def strip_prompt_login(path):
+        """
+        Strips 'login' from the 'prompt' query parameter.
+        """
+        uri = urlsplit(path)
+        query_params = parse_qs(uri.query)
+        prompt = query_params['prompt']
```

Comment:
Oh yeah. Thanks for spotting this. The if was meant to find the case where prompt is empty after removing login. If prompt was empty it wouldn't need to be appended to the URL. Will fix.